### ISSUE TITLE
automatic page scrolling with NVDA

### ISSUE THREAD

**Issue by AntonioCezar** [mush42/bookworm#26]
Hello. I have to read numerous PDF files because of my law school, and for me, the Bookworm is really great in the PDF files open quickly and also in other resources you have. But I lack a feature that I consider really essential: The Bookworm does not scroll the pages automatically when I'm reading with NVDA, the way it promises to do when we are using the internal mechanism of the application speech. I know I can hit enter when the cursor reaches the end of a page, but the automatic page scrolling would be much more comfortable, especially for a continuous reading of very large books, such as the Laws of books. And besides, I may have to copy a paragraph that starts on a page and ends in another. Anyway, I would like to bring the request for more this important resource for the community and especially to the developer, whom I thank for the excellent work and effort in bringing a good PDF reader accessible to the blind, and I hope to have the support of all ... Thank you.

**Comment by mush42** [comment]
Closing as it is a duplicate of #3.

**Comment by mush42** [comment]
Hi @AntonioCezar 
Please follow the discussion at issue number #3 .
Regards,
